2021_12_12_103026_alter_imobils_table_add_seria_column.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class AlterImobilsTableAddSeriaColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+       Schema::table('imobils', function (Blueprint $table) {
+           $table->string('seria', 40)->default('143')
+           ->after('balcony');
+
+
+       });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```

Comment:
rollback is missing

Reply:
put rollback
Schema::table('imobils', function (Blueprint $table) {
                $table->dropColumn('seria');
        });